docs/docs/http-api.mdx

```diff
@@ -66,6 +66,21 @@ rasa run \
     --auth-token thisismysecret
 ```
 
+or via environment variable:
```

Comment:
Do we need to update the CLI docs too?

Reply:
We definitely can. I will include @ArjaanBuijk to have someone from the CSE team to help us confirming if this would be the right approach.